finality-aleph/src/sync/ticker.rs

```diff
@@ -1,39 +1,37 @@
 use tokio::time::{sleep, Duration, Instant};
 
 /// This struct is used for rate limiting as an on-demand ticker. It can be used for ticking
-/// at least once `max_timeout` but not more than once every `min_timeout`.
-/// Example usage would be to use `wait` method in main select loop and `try_tick` whenever
-/// you would like to tick sooner in another branch of select.
+/// at most after `max_timeout` but no sooner than after `min_timeout`.
+/// It will always tick after `max_timeout` and never after that unless reset earlier.
```

Comment:
```suggestion
/// It will always tick before `max_timeout` and never after that unless reset earlier.
```
?

Reply:
Ugh, that's also not what I meant, but trying to write what I meant turned out to bee too hard, so I just deleted this line.

The documentation next to the methods should be good enough.

Reply:
Yep, it was confusing as hell :P